library/src/test/java/com/ironz/binaryprefs/encryption/XorKeyEncryptionImplTest.java

```diff
@@ -0,0 +1,35 @@
+package com.ironz.binaryprefs.encryption;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+/**
+ * @author Sergey Boishtyan
+ */
+public class XorKeyEncryptionImplTest {
```

Comment:
Also you should check `checkMirror` method with even size, non-even size with non-mirrored byte array and the same with mirrored byte array.